src/metamorpho/MetaMorphoSnippets.sol

```diff
@@ -177,6 +183,41 @@ contract MetaMorphoSnippets {
         redeemed = IMetaMorpho(vault).redeem(maxToRedeem, receiver, msg.sender);
     }
 
+    /// @notice Withdraws the maximum available liquidity from the markets `srcMarketParams`, and supply the withdrawn
+    /// assets into destMarketParams.
+    /// @dev The snippets contract must be registered as an allocator of the vault for the fuction to be called
+    /// successfully.
+    /// @param vault The address of the MetaMorpho vault.
+    /// @param srcMarketParams The market parameters of the markets to withdraw from.
```

Comment:
this is a list of marketParams that is passed as argument so the comment is not accurate